src/test/java/de/refactoringbot/refactorings/RenameMethodTest.java

```diff
@@ -53,7 +52,7 @@ public void testRenameMethod() throws Exception {
 		String originalSecondMethodName = RefactoringHelper
 				.getMethodByLineNumberOfMethodName(lineNumberOfSecondMethodNotToBeRenamed, cuOriginalFile)
 				.getNameAsString();
-		assertEquals(originalMethodName, originalSecondMethodName);
+		assertThat(originalSecondMethodName).isEqualTo(originalMethodName);
```

Comment:
could also use SoftAssertions for this method.

Reply:
In this method, I'm not sure how to use SoftAssertions in a way that makes sense and doesn't reduce the readability of the code. On the one hand I suspect that `notNotNull` assertions in this example are not used with SoftAssertions, because otherwise the following code could throw a NullPointerException and stop execution. Then again, I find it confusing if a SoftAssertion stretches over a large block of code that also contains normal assertions. Can you give me some more feedback here? This is my first contact with SoftAssertions :)

Reply:
Right, here it does not really add a benefit, because of the notNull assertions. You can leave it as is.